qp_qiime2/__init__.py

```diff
@@ -21,176 +25,179 @@
 
 # Define the rarefy command
 req_params = {
-    'i-table': ('artifact', ['BIOM']),
-    'p-sampling-depth': ['integer', 1000]
+    'BIOM table': ('artifact', ['BIOM']),
+    'Sampling depth': ['integer', 1000]
 }
 opt_params = {}
-outputs = {'o-table': 'BIOM'}
+outputs = {'Rarefied table': 'BIOM'}
 dflt_param_set = {
     'Defaults': {
-        'p-sampling-depth': 1000}
+        'Sampling depth': 1000}
 }
 qiime_cmd = QiitaCommand(
-    "Rarefy", "Rarefy",
+    "Rarefy features", "Rarefy",
     rarefy, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the beta_diversity command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {
-    'p-metric': [
-        ('choice:["sokalsneath", "kulsinski", "mahalanobis", "matching", '
-         '"euclidean", "correlation", "yule", "russellrao", "hamming", '
-         '"jaccard", "braycurtis", "dice", "rogerstanimoto", "sqeuclidean", '
-         '"cityblock", "sokalmichener", "cosine", "wminkowski", "seuclidean", '
-         '"chebyshev", "canberra", "unweighted UniFrac", '
-         '"weighted normalized UniFrac", "weighted unnormalized UniFrac"]'),
-        'jaccard'],
-    'i-tree': ['choice:["default", "None"]', 'None']}
-outputs = {'distance_matrix': 'distance_matrix'}
+    'Diversity metric': [
+        'choice:%s' % dumps(list(BETA_DIVERSITY_METRICS)),
+        'Jaccard similarity index'],
+    'Phylogenetic tree': ['choice:["default", "None"]', 'None'],
+    'Number of jobs': ['integer', 1],
+    'Adjust variance (phylogenetic only)': ['boolean', False],
+    'Alpha value (Generalized Unifrac only)': ['float', 0],
+    'Bypass tips (phylogenetic only)': ['boolean', False]}
+outputs = {'Distance matrix': 'distance_matrix'}
 dflt_param_set = {
     'Defaults': {
-        'p-metric': 'jaccard',
-        'i-tree': 'None'}
+        'Diversity metric': 'Jaccard similarity index',
+        'Phylogenetic tree': 'None',
+        'Number of jobs': 1,
+        'Adjust variance (phylogenetic only)': False,
+        'Alpha value (Generalized Unifrac only)': 0,
+        'Bypass tips (phylogenetic only)': False}
 }
 qiime_cmd = QiitaCommand(
-    "beta_diversity", "Beta Diversity",
+    "Calculate beta diversity", "Beta Diversity",
     beta_diversity, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the pcoa command
-req_params = {'i-distance-matrix': ('artifact', ['distance_matrix'])}
+req_params = {'Distance matrix': ('artifact', ['distance_matrix'])}
 opt_params = {}
-outputs = {'o-pcoa': 'ordination_results'}
+outputs = {'Ordination results': 'ordination_results'}
 dflt_param_set = {
     'Defaults': {}
 }
 qiime_cmd = QiitaCommand(
-    "pcoa", "Principal Coordinate Analysis",
+    "Perform Principal Coordinates Analysis (PCoA)",
```

Comment:
This is the only one that is Perform vs. Calculate and the only one that has capital initial letters, fine if we want to leave as this.

Reply:
That was changed as request by @wasade to match the capitalization of PCoA

Reply:
ok